### PR TITLE
Contact box starting page and support

### PR DESCRIPTION
**Pull request by kress04**
Added contact info underneath news in index.md & support.md. Additional css wrapper.

Not optimized for mobile view. As we are going to migrate to Wordpress shortly and almost 95% of visits are from desktop, I saved efforts for optimization.

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -10,16 +10,24 @@ nav:
   bottom: false
 ---
 
+<div class="wrapper">
+  <div class="latest-news">
+	  <h4>Important contact information</h4>
+	If you're having technical questions please use the <a href="https://board.karakun.com/viewforum.php?f=21" target="_blank">community support forum</a>. Customers with a maintenance contract should use <a href="https://board.karakun.com/viewforum.php?f=5" target="_blank">Premium Support forum</a>.<br/><br/>
```
Comment (janakmulani) [inline support.md:16]: Customers with Premium Support contract should use ....

File: support.md [hunk 2]
```diff
@@ -10,16 +10,24 @@ nav:
...
+	For commercial questions please write an email at <a href="mailto:openwebstart@karakun.com">openwebstart@karakun.com</a>. Please do not post technical questions to this email, use the forum instead.
+	</div>
+</div>
+
 # OWS Support Forum
 
 The most easiest way is our new support forum. Visit [https://board.karakun.com](https://board.karakun.com) to access the forum.
```
Comment (janakmulani) [inline support.md:23]: The easiest way is our ...

File: support.md (hunk 1 above)
Comment (kress04) [reply support.md]: I added "Customers with Basic or Premium support contract..." as we are addressing both groups, aren't we?

File: support.md (hunk 2 above)
Comment (kress04) [reply support.md]: done.

File: support.md (hunk 1 above)
Comment (janakmulani) [reply support.md]: ok

**Pull request merged**